yogstation/code/modules/client/verbs/afk.dm

```diff
@@ -89,6 +89,9 @@
 			adminhelp("I need to go AFK as '[important_role]' for duration of '[time]' [reason ? " with the reason: '[reason]'" : ""]")
 			mob.log_message("is now AFK for [time] [reason ? " with the reason: '[reason]'" : ""]", LOG_OWNERSHIP)
 		else
-			to_chat(src, "<span class='danger'>Admins will not be automatically alerted, because you do not seem to be in a critical station role.</span>")
+			to_chat(src, "<span class='danger'>Admins will be subtly alerted, because you do not seem to be in a critical station role.</span>")
+			var/normal_role = special_role || M.job || initial(M.name) || "unknown job"
+			message_admins("[ADMIN_LOOKUPFLW(src)] has used the AFK verb as '[normal_role]' for duration of '[time]'")
+			log_admin("[key_name(src)] has used the AFK verb as '[normal_role]' for duration of '[time]'")
```

Comment:
log_game or log_admin_private, take your pick.
log_admin is for admin actions

log_game is used for disconnects/connects/suicides so it feels pretty fitting.
log_admin_private is used to tickets so that's an alternative too.

Don't do both.